docs(linux): AM62X: add release note for AM62SIP support (#360)
TexasInstruments/processor-sdk-doc: source/devices/AM62X/linux/Release_Specific_Release_Notes.rst

**Processor SDK Linux AM62X Release has following new features:**

  - Second 2025 LTS Reference Release Including RT combined branch model

>   - Add support for AM62SIP, which can be taken as an example of using AM62x with 512MiB of RAM

Reviewer: ```suggestion
  - Add support for AM62SIP, which is an example of using AM62x with 512MiB of RAM
```

Author: Updated in [cefc634](https://github.com/TexasInstruments/processor-sdk-doc/commit/cefc6344864ce740d7d90e8ae886e5bbeb76260e). Thanks!
